dlc-wallet-test/src/test/scala/org/bitcoins/dlc/wallet/DLCMultiOracleNumericExecutionTest.scala

```diff
@@ -249,4 +212,32 @@ class DLCMultiOracleNumericExecutionTest
         aggregateSignature == statusB.oracleSig
     }
   }
+
+  /** Builds an attestment for the given numeric oracle outcome */
+  private def buildAttestments(
```

Comment:
probably would be nice to have this in one place instead of multiple (also my above comments apply here)